### PR TITLE
Fix Issue 17262 - Better docs for rdmd

### PR DESCRIPTION
**Pull request by wilzbach**
We should really auto-generate this like e.g. DUB does ...

### CODE REVIEW

**Comment by wilzbach** [comment]
Respective PR at the tools repo: https://github.com/dlang/tools/pull/232

File: rdmd.dd [hunk 1]
```diff
@@ -110,6 +114,8 @@ $(DT $(B --makedepend)) $(DD print dependencies in makefile format and exit)
 $(DT $(B --man)) $(DD open web browser on manual page)
 
 $(DT $(B --shebang)) $(DD rdmd is in a shebang line (put as first argument))
+
+$(DT $(B --tempdir)) $(DD use a different temporary directory)
```
Comment (CyberShadow) [inline rdmd.dd]: It`s `--tmpdir`. It's not in `--help` though.

**Comment by wilzbach** [comment]
> It`s `--tmpdir`. It's not in `--help` though.

Argh. Thanks!

-> https://github.com/dlang/tools/pull/232/files

**Comment by CyberShadow** [comment]
Travis stuck again.

**Comment by d-random-contributor** [comment]
Better formulation:
>By default, the executable is placed in the source file directory

**Comment by wilzbach** [comment]
> By default, the executable is placed in the source file directory

Good idea. Thanks!

**Pull request merged**